portfolio/src/main/webapp/index.html

```diff
@@ -51,5 +52,72 @@ <h3 class="project-name">Flappy Bird</h3>
             <p class="tech-used">Technology Used: Python and the Pygame module.</p>
         </div>
       </div>
+      <div id="content">
+        <h2 class="header" id="gallery">Gallery</h2>
+        <p>This gallery consists of my favorite places and things in life. When I take pictures, I prefer them to be memories of the locations I have experienced and the places that I have seen.</p>
+        <p>Although none of these images are of my own, each one represents a part of me that I truly cherish and favor. I mostly enjoy aesthetic scenary or locations of places that I would like to go or have personally been to.</p>
+        <div class="gallery">
+            <a target="_blank" href="/images/japan3.jpg">
+                <img src="/images/japan3.jpg" alt="Japanese Sunset Scenary" width="600" height="400">
```

Comment:
Optional: Since width and height is defined in gallary class, you can remove those attributes here.